avalon/pipeline.py

```diff
@@ -967,6 +956,9 @@ def update_current_task(task=None, asset=None, app=None):
     _session = Session.copy()
     _session.update(changed)
     changed["AVALON_WORKDIR"] = _format_work_template(template, _session)
+    changed['AVALON_HIERARCHY'] = os.path.sep.join(
+            asset_document['data']['parents']
```

Comment:
This will break pipelines that don't have the `parents` data, right? This should allow for backwards compatibility.

Reply:
true. we'll put a .get() there with default of empty string to make sure it always resolve to something.